dataikuapi/dss/ml.py

```diff
@@ -801,6 +1147,22 @@ def get_performance_metrics(self):
                 del clean_snippet[x]
         return clean_snippet
 
+    @property
+    def assertions_metrics(self):
+        """
+        Retrieves assertions metrics computed for this trained model
+        :returns: an object representing assertion metrics
+        :rtype: :class:`dataikuapi.dss.ml.DSSMLAssertionsMetrics`
+        """
+        return self.get_assertions_metrics()
```

Comment:
Not sure a property helps if you have a no-argument getter already. Seems like just an additional thing to maintain. (Saw it elsewhere too).

Reply:
I find it a bit nicer to write `model_details.assertions_metrics.get_per_assertion_metric("women")` than `model_details.get_assertions_metrics().get_per_assertion_metric("women")`. Chaining getters seems uglier. But I am happy to change if it is not worth the maintenance cost

Reply:
I don't think it is indeed. I'm ok on "terminal" objects to easily get/set a direct value, but everywhere else where we get complex/custom objects, we have getters…